core-java-modules/core-java-date-operations-3/src/test/java/com/baeldung/unixtime/UnixTimeUtilsUnitTest.java

```diff
@@ -0,0 +1,106 @@
+package com.baeldung.unixtime;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.Assert.assertEquals;
+
+import java.time.Instant;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoField;
+import java.util.Calendar;
+import java.util.Date;
+
+import org.junit.jupiter.api.Test;
+
+public class UnixTimeUtilsUnitTest {
+    private static final String AUG_16_2022_15h25m32_Z_FORMATTED = "2022-08-16T15:25:32";
+    private static final long AUG_16_2022_15h25m32_Z_NANOS = 1660663532747420283l;
+    private static final long AUGUST = 8;
+
+    private void assertInstantFieldsMatch(LocalDateTime time) {
+        assertEquals(AUGUST, time.get(ChronoField.MONTH_OF_YEAR));
+        assertEquals(16, time.get(ChronoField.DAY_OF_MONTH));
+        assertEquals(2022, time.get(ChronoField.YEAR));
+        assertEquals(15, time.get(ChronoField.HOUR_OF_DAY));
+        assertEquals(25, time.get(ChronoField.MINUTE_OF_HOUR));
+        assertEquals(32, time.get(ChronoField.SECOND_OF_MINUTE));
```

Comment:
Instead of using org.junit.Assert.assertEquals from JUnit 4, could you use assertEquals of org.junit.jupiter.api.Assertions instead?